### PR TITLE
A Seqr management command testcase

### PR DESCRIPTION
**Pull request by ShifaSZ**
The first testcase for testing the normal functionality of the command add_project_tag.

### CODE REVIEW

File: seqr/management/commands/add_project_tag_tests.py [hunk 1]
```diff
@@ -17,11 +21,62 @@ class AddProjectTagTest(TestCase):
...
+        call_command('add_project_tag', '--project={}'.format(TAG_ARGUMENTS["project"]),
+            '--name={}'.format(TAG_ARGUMENTS["name"]), '--order={}'.format(TAG_ARGUMENTS["order"]),
+            '--category={}'.format(TAG_ARGUMENTS["category"]),
+            '--description={}'.format(TAG_ARGUMENTS["description"]),
+            '--color={}'.format(TAG_ARGUMENTS["color"]), stdout = out)
+
+        self.assertIn('', out.getvalue())
+
+        project = Project.objects.get(Q(name = TAG_ARGUMENTS["project"]))
+        variantTagType = VariantTagType.objects.get(name__iexact=TAG_ARGUMENTS["name"], project=project)
```
Comment (hanars) [inline seqr/management/commands/add_project_tag_tests.py:33]: theres no reason for `iexact` here, you can just do `name=`

File: seqr/management/commands/add_project_tag_tests.py [hunk 2]
```diff
@@ -17,11 +21,62 @@ class AddProjectTagTest(TestCase):
...
-
-        expectOutput = out.getvalue()
-        self.assertIn('', expectOutput)
+        call_command('add_project_tag', '--project={}'.format(TAG_ARGUMENTS["project"]),
+            '--name={}'.format(TAG_ARGUMENTS["name"]), '--order={}'.format(TAG_ARGUMENTS["order"]),
+            '--category={}'.format(TAG_ARGUMENTS["category"]),
+            '--description={}'.format(TAG_ARGUMENTS["description"]),
+            '--color={}'.format(TAG_ARGUMENTS["color"]), stdout = out)
+
+        self.assertIn('', out.getvalue())
```
Comment (hanars) [inline seqr/management/commands/add_project_tag_tests.py:30]: All this test is that some string contains an empty string, which is always true for all strings in python. If theres no output you could just skip testing the output altogether, or if you want to check that there is no output you should use `asserEqual`

File: seqr/management/commands/add_project_tag_tests.py [hunk 3]
```diff
@@ -17,11 +21,62 @@ class AddProjectTagTest(TestCase):
...
+
+    def test_missing_required_args(self):
+        out = StringIO()
+        error = ''
+        try:
+            call_command('add_project_tag',
+                stdout = out)
+        except CommandError as err:
+            error = err.message
+        self.assertEqual(error, 'Error: argument --project is required')
```
Comment (hanars) [inline seqr/management/commands/add_project_tag_tests.py:52]: If you want to unit test exceptions you should use `assertRaises`: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises

File: seqr/management/commands/add_project_tag_tests.py (hunk 1 above)
Comment (ShifaSZ) [reply seqr/management/commands/add_project_tag_tests.py:33]: Okay. It was from a copy-paste. Remove it.

File: seqr/management/commands/add_project_tag_tests.py (hunk 2 above)
Comment (ShifaSZ) [reply seqr/management/commands/add_project_tag_tests.py:30]: Oh, really?! Change it to assertEqual.

File: seqr/management/commands/add_project_tag_tests.py (hunk 3 above)
Comment (ShifaSZ) [reply seqr/management/commands/add_project_tag_tests.py]: Hmm... I remember it now. It is easy to forget if never use it. Changed to use assertRaises.

**Pull request merged**